dir.props

```diff
@@ -263,10 +263,12 @@
     <DotnetHostString>dotnet-host-</DotnetHostString>
     <DotnetHostFxrString>dotnet-hostfxr-</DotnetHostFxrString>
     <DotnetRuntimeString>dotnet-runtime-</DotnetRuntimeString>
+    <DotnetRuntimeMetaPackageString>dotnet-runtime-dep-</DotnetRuntimeMetaPackageString>
```

Comment:
The Docker container uses the term `runtime-deps`.  We should align here and add a `s`.  /cc @michaelsimons

Reply:
I agree - `runtime-deps` - see https://github.com/dotnet/dotnet-docker/blob/master/README.md#L11